content/en/cosign/openid_signing.md

```diff
@@ -35,6 +35,26 @@ To log in and set up your OIDC Identity, follow these steps:
 
 Currently, Sigstore supports Microsoft, Google, and GitHub. As an alternative, you can also use the environment variable `SIGSTORE_ID_TOKEN` to identify yourself by setting its value to that of the identity token. Cosign also has support for detecting some of these automated environments and producing an identity token. Currently this supports Google and GitHub, in addition to other environments. See [Cosign's providers](https://github.com/sigstore/cosign/tree/main/pkg/providers) for a complete list.
 
+### The signing, witnessing, and verifying process
+
+When taking advantage of sigstore’s full capabilities, the process of signing is as follows:
+
+#### Verifying identity and signing the artifact
+
+1) A keypair is created. 
+2) Sigstore checks the identity of the user signing the artifact and issues a certificate attesting to their identity. 
+3) The certificate is bound to the public key, which adds an extra layer of verification. Exchanging the keys will prove the identity of the private keyholder. 
```

Comment:
What do you mean by "exchanging the keys"?

Reply:
I wasn't sure what this meant either.   Does this mean the public key is somehow exchanged with a symmetric key, or is this sentence simply wrong?